server/routes/overview/mappers/functionalSkillsMapper.ts

```diff
@@ -28,6 +28,12 @@ const dateOrNull = (value: string): Date | undefined => {
   return value ? startOfDay(parseISO(value)) : undefined
 }
 
+const formatPrisonName = (prisonName: string) => {
```

Comment:
No, why is the mapper formatting the prison name? We dont want to do this here (formatting the prison name is not always as easy as lower casing / upper casing as you have here). This is why we do a lookup from the prison-register API